.github/workflows/release.yaml

```diff
@@ -90,6 +90,13 @@ jobs:
         run: |
           docker push kubeshop/testkube-cli:${{ steps.github_sha.outputs.sha_short }}-arm64v8
           docker push kubeshop/testkube-cli:${{ steps.github_sha.outputs.sha_short }}-amd64
+          # adding the docker manifest for the latest image tag
```

Comment:
This should be handled in `.builds-linux.goreleaser.yml` and this whole step can be removed.

We already build and push images in the previous step when running `goreleaser release`, this pushes the same images again.

Here is the respective GoReleaser config https://github.com/kubeshop/testkube/blob/develop/.builds-linux.goreleaser.yml#L56-L81